ruleset/sca/centos/7/cis_centos7_linux.yml

```diff
@@ -42,6 +45,7 @@ checks:
       - cis_csc: ["5.1"]
       - pci_dss: ["2.2.5"]
       - tsc: ["CC6.3"]
+      - cis_level: ["1"]
```

Comment:
Can you explain where do this field and values come from?
Thank you!

Reply:
From the official "Center for Internet Security CentOS 7 Benchmark v3.0.0 - 06-30-2020" documentation that was used to create this rule file (That document is also mentioned in the top of the rules file). Not all companies need/want the cis Level 2 scores in the documentation. Often they start with the Level 1 rules, or only need the Level 1 rules in the documentation and later on move to the Level 2 rules if needed.  This way it is visible in the events if it is a Level 1 or Level 2 event.